fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -1966,4 +1966,34 @@ public void testPutReferenceRoot() throws Exception {
         }
     }
 
-}
\ No newline at end of file
+    @Test
+    public void inboundLinksDoNotUpdateEtag() throws IOException {
```

Comment:
It is minor, but the naming convention is to begin the test methods with "test", such as: `testInboundLinksDoNotUpdateEtag()`